docs/src/main/sphinx/release/release-358.md

```diff
@@ -0,0 +1,40 @@
+# Release 358 (XX May 2021)
+
+## General
+
+* Support any query in {doc}`/sql/show-stats`. ({issue}`8026`)
+* Improve performance of complex queries involving joins and `TABLESAMPLE`. ({issue}`8094`)
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data. ({issue}`6634`)
+* Reduce worker graceful shutdown time. ({issue}`8149`)
```

Comment:
Reduce graceful shutdown time for workers.

or 

Reduce graceful shutdown time for worker nodes.

Reply:
Worth indicating whether this means reduced guarantees (eg smaller timeout value), or better logic.